src/PAModel/Schemas/PcfControls/Property.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.PowerPlatform.Formulas.Tools.Schemas
```

Comment:
nit: maybe these should have the namespace Microsoft.PowerPlatform.Formulas.Tools.Schemas.PcfControl

Property is a pretty generic concept otherwise.